charts/appswitcher-server/templates/configmap.yaml

```
@@ -2,9 +2,9 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "appswitcher-server-helm-chart.fullname" . }}
+  name: {{ include "common.fullname" . }}
   labels: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)